packages/perseus/src/widgets/radio/choice-icon/choice-icon.jsx

```diff
@@ -1,15 +1,20 @@
 // @flow
-/* Component for rendering a letter icon in radio choice */
+/* Component for rendering a letter icon in a library radio choice */
 
 import Color from "@khanacademy/wonder-blocks-color";
+import {StyleSheet, css} from "aphrodite";
 import * as React from "react";
 
+import InlineIcon from "../../../components/inline-icon.jsx";
+import {iconCheck, iconMinus} from "../../../icon-paths.js";
+import * as styleConstants from "../../../styles/constants.js";
+import FocusRing from "../focus-ring.jsx";
 import {getChoiceLetter} from "../util.js";
 
-import LibraryChoiceIcon from "./library-choice-icon.jsx";
-import SATChoiceIcon from "./sat-choice-icon.jsx";
+import CrossOutLine from "./cross-out-line.jsx";
+import sharedStyles, {LIBRARY_ICON_SIZE} from "./shared-styles.js";
```

Comment:
Just a nit, but I wonder if it'd be worth changing this to `CHOICE_ICON_SIZE` now that we're not trying to distinguish between library and SAT